react/features/base/conference/reducer.js

```diff
@@ -161,13 +162,19 @@ function _conferenceFailed(state, { conference, error }) {
     }
 
     let authRequired;
+    let membersOnly;
     let passwordRequired;
 
     switch (error.name) {
     case JitsiConferenceErrors.AUTHENTICATION_REQUIRED:
         authRequired = conference;
         break;
 
+    case JitsiConferenceErrors.CONFERENCE_ACCESS_DENIED:
```

Comment:
why is there no difference between these 2 errors?

Reply:
both errors are related to the lobby mode, they are just different stage of the flow